finagle-netty4/src/main/scala/com/twitter/finagle/netty4/channel/ServerBridge.scala

```diff
@@ -15,6 +17,17 @@ private[netty4] class ServerBridge[In, Out](
 
   def initChannel(ch: Channel): Unit = {
     val transport: Transport[In, Out] = transportFac(ch)
-    serveTransport(transport)
```

Comment:
Not really, if I understand your question correctly. `initChannel` will be called if `channelRegistered` event is triggered or a `ChannelInitializer` is added to a pipeline of a registered channel.